auth.proto

```diff
@@ -13,9 +13,12 @@ service ThingsService {
     rpc Identify(Token) returns (ThingID) {}
 }
 
-service AuthNService {
+service AuthService {
     rpc Issue(IssueReq) returns (Token) {}
```

Comment:
Since Auth service now works with three entity types (Keys, Groups, Policies), we should probably be more specific here: `IssueKey` or `IssueToken`.